newhelm/runners/simple_benchmark_runner.py

```diff
@@ -24,31 +24,38 @@ def __init__(self, data_dir: str, max_test_items: Optional[int] = None):
         self.data_dir = data_dir
         self.max_test_items = max_test_items
 
-    def run(self, benchmark: BaseBenchmark, suts: List[SUT]) -> List[BenchmarkRecord]:
+    def run(
+        self, benchmark: BaseBenchmark, suts: Mapping[str, SUT]
+    ) -> List[BenchmarkRecord]:
         # Not all runners can run all Test types, so validate up front
-        prompt_response_tests: List[BasePromptResponseTest] = []
-        for test in benchmark.get_tests():
+        prompt_response_tests: Dict[str, BasePromptResponseTest] = {}
+        for test_name, test in benchmark.get_tests().items():
             if isinstance(test, BasePromptResponseTest):
-                prompt_response_tests.append(test)
+                prompt_response_tests[test_name] = test
             else:
-                raise Exception("Runner can't handle test:", test.__class__.__name__)
+                raise Exception("Runner can't handle test:", test_name)
```

Comment:
For errors, I'd want both the test name and a class name, as that will make it clearer what went wrong.

Reply:
Agreed. Made it include both `test_name` and `test.__class__`.